packages/stateful/voting-module-adapter/adapters/CwdVotingCw721Staked/daoCreation/GovernanceConfigurationReview.tsx

```diff
@@ -0,0 +1,72 @@
+import { useTranslation } from 'react-i18next'
+import { constSelector, useRecoilValueLoadable } from 'recoil'
+
+import { Cw721BaseSelectors } from '@dao-dao/state'
+import {
+  CopyToClipboard,
+  DaoCreateVotingPowerDistributionReviewCard,
+  FormattedJsonDisplay,
+} from '@dao-dao/stateless'
+import { DaoCreationGovernanceConfigReviewProps } from '@dao-dao/types'
+
+import { DaoCreationConfig, GovernanceTokenType } from '../types'
+
+export const GovernanceConfigurationReview = ({
+  data: { tokenType, existingGovernanceTokenAddress },
+}: DaoCreationGovernanceConfigReviewProps<DaoCreationConfig>) => {
+  const { t } = useTranslation()
+
+  const existingGovernanceTokenInfoLoadable = useRecoilValueLoadable(
+    tokenType === GovernanceTokenType.Existing && existingGovernanceTokenAddress
+      ? Cw721BaseSelectors.contractInfoSelector({
+          contractAddress: existingGovernanceTokenAddress,
+          params: [],
+        })
+      : constSelector(undefined)
+  )
+
+  const numOfTokensLoadable = useRecoilValueLoadable(
+    tokenType === GovernanceTokenType.Existing && existingGovernanceTokenAddress
+      ? Cw721BaseSelectors.numTokensSelector({
+          contractAddress: existingGovernanceTokenAddress,
+          params: [],
+        })
+      : constSelector(undefined)
+  )
+
+  // If existing token, just display the token info again since there are no
+  // tier distributions to display.
+  if (tokenType === GovernanceTokenType.Existing) {
```

Comment:
can remove this check and the placeholder at the bottom once removing the token type and new/existing distinction from this adapter